server/app/services/program/ProgramDefinition.java

```diff
@@ -500,10 +533,13 @@ public ImmutableList<BlockDefinition> getNonRepeatedBlockDefinitions() {
    *   <li>Is not an enumerator.
    * </ul>
    */
-  public ImmutableList<QuestionDefinition> getAvailablePredicateQuestionDefinitions(long blockId)
+  private ImmutableList<QuestionDefinition> getAvailablePredicateQuestionDefinitions(long blockId)
```

Comment:
suggestion (blocking): looks like the javadoc for this isn't quite accurate after this change --> "In a block definition that comes sequentially before the given block definition."

Reply:
thanks, c&p error.

I thought it better to make this private which made the doc not technically reference-able in the other methods docs